src/ui.js

```diff
@@ -502,6 +502,18 @@ UIMode.Command = class extends UIMode {
     }
 };
 
+// We are viewing a diagram embedded in another page.
+UIMode.Embed = class extends UIMode {
+    constructor(ui) {
+        super();
+
+        this.name = "embed";
+
+        ui.grid.class_list.toggle("hidden");
+        ui.pan_view(Offset.zero(), -1.0);
+    }
```

Comment:
Can you change the **quiver** logo `href` in this case to be the URL to the diagram in [quiver.app](https://q.uiver.app/)?